lib/src/rules/always_declare_return_types.dart

```diff
@@ -66,13 +66,13 @@ class AlwaysDeclareReturnTypes extends LintRule {
 class _Visitor extends SimpleAstVisitor<void> {
   static const LintCode functionCode = LintCode(
       "always_declare_return_types", // ignore: prefer_single_quotes
```

Comment:
We could also drop these ignore comments at some point. Not required for this PR, of course.